newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -119,6 +119,23 @@ export const GameDetailsDialog = ({
     setIsPublicGamePropertiesDialogOpen,
   ] = React.useState(false);
 
+  // If a game dashboard tab is specified, switch to it.
+  React.useEffect(
+    () => {
+      if (routeArguments['games-dashboard-tab']) {
```

Comment:
I wonder if this string should be stored in a variable such as:
```js
ROUTING_ARGUMENTS = {
  gamesDashboardTab: 'games-dashboard-tab',
  ...
}
```

Reply:
yeah I'm afraid this will make the code and imports heavier 🤔

Reply:
I've improved the code to use proper route types, avoiding those magic strings